### PR TITLE
nokogiri (>= 1.10.4) (security alert)

### PR DESCRIPTION
**Pull request by makyen**
Change nokogiri to >=1.10.4, due to the security alert [here](https://github.com/Charcoal-SE/metasmoke/network/alert/Gemfile.lock/nokogiri/open).

### CODE REVIEW

**Comment by makyen** [comment]
Thanks for telling me. I should have noticed that from its format.

Frankly, I *really* should have been more thorough in looking at this issue. Sorry about that.

`nokogiri` is used three times in [`Gemfile.lock`](https://github.com/Charcoal-SE/metasmoke/blob/master/Gemfile.lock), stating it's a dependency for two packages: `rails-dom-testing` and `loofah`, and has a dependency itself, `mini_portile2`.  The [`Gemfile`](https://github.com/Charcoal-SE/metasmoke/blob/master/Gemfile) doesn't have an entry for `nokogiri`.

Naively, it *may* be possible to just add the `nokogiri` requirement to `Gemfile`, but we're beyond what I'd want to do without an environment in which I can build and fully test (not just the automated tests) on a machine/environment where I could run a version of the site w/o impacting other people.

At a minimum, a new `Gemfile.lock` would need to be generated through the build process. In a branch, I could shoehorn the file to be displayed by CircleCI and copy-&-paste what CircleCI generates, but I'd really want to have an actual full-up test setup to verify that there aren't related dependency issues, or that nothing actually broke through the adjustment of some related dependancy. So, doing this one correctly is beyond the environment I have set up at this time.

Basically, prior experience with dependency issues makes me hesitant to move forward at this time. So, I'm just going to close this and let one of you handle it.

**Comment by thesecretmaster** [comment]
If you're OK with just updating everything (which should be fine) then you can just run `bundle update` and that will generate a new Gemfile.lock with updated dependencies.

**Comment by ArtOfCode-** [comment]
I'll get to this when I've got some time at a dev machine next, no worries
